guides/common/modules/con_resolving-package-dependencies.adoc

```diff
@@ -3,16 +3,48 @@
-The package dependency resolution feature does not consider packages that are installed on your system independently of the Content View or solve dependencies across repositories.
+Note that resolving package dependencies can cause significant delays to Content View publication.
+The more content and repositories in the content view, the greater the delay.
+
+The package dependency resolution feature does not take which packages are installed on your systems into account.
+Only packages within the content view's repositories are considered.
+For example, if a content view has only AppStream in it, dependency solving will not know to pull dependent BaseOS content in at publish time.
+For more information, see {ContentManagementDocURL}Limitations_to_Repository_Dependency_Resolution_content-management[Limitations to Repository Dependency Resolution] in _{ContentManagementDocTitle}_.
 
 .Resolving Package Dependencies and Filters
 
-Filters do not resolve any dependencies of the packages listed in the filters.
+Filters, by default, do not resolve any dependencies of the packages listed in the filters.
 This might require some level of testing to determine what dependencies are required.
 
 If you add a filter that excludes some packages that are required and the Content View has dependency resolution enabled, {Project} ignores the rules you create in your filter in favor of resolving the package dependency.
 
-If you create a content filter for security purposes, to resolve a package dependency, {Project} can add packages that you might consider insecure.
+If you create a content filter for security purposes but choose to resolve dependencies, {Project} can add packages that you might consider insecure.
+
+.Filtering and Dependency Solving Recommendations
+
+Be mindful when combining exclusion filters with dependency solving.
+For example, take a scenario where a user wants to recreate RHEL 8.3 via content view filters and include select packages and errata from a later RHEL 8 y-release.
+The user excludes all errata after the RHEL 8.3 release date minus some select few that are needed and enables dependency solving.
+Dependency solving may pull in more RPMs than expected, which would cause the system to stray further from being a RHEL 8.3 machine.
+If dependency solving pulls in too many unwanted RPMs, consider identifying by hand the core basic dependencies that the extra packages and errata need.
+
+If, in the above case, the user does not need the extra errata and RPMs, then content view filtering is not needed at all.
+The RHEL 8.3 repository could be enabled and used directly from the Red Hat Repositories page.
+
+In some cases, excluding RPMs may make dependency solving impossible.
+For example, if a user wants to make a RHEL 8.3 repo with a few RPMs missing, dependency solving should not be used because the missing RPMs could be dependencies for another RPMs in the repository.
```

Comment:
```suggestion
For example, if a user wants to make a RHEL 8.3 repo with a few RPMs missing, dependency solving should not be used because the missing RPMs could be dependencies for other RPMs in the repository.
```